tests/test_wave_01.py

```diff
@@ -51,22 +51,26 @@ def test_get_task(client, one_task):
     }
 
 
-@pytest.mark.skip(reason="No way to test this feature yet")
+# @pytest.mark.skip(reason="No way to test this feature yet")
 def test_get_task_not_found(client):
     # Act
     response = client.get("/tasks/1")
     response_body = response.get_json()
 
     # Assert
     assert response.status_code == 404
+    assert response_body == {
+        "details": "Id not found"
+    }
+    assert Task.query.all() == []
```

Comment:
Generally, we should try to include as much detail in our error message. An improvement can include adding the exact id that was not found. 

```suggestion
    assert response_body == {
        "details": "id 1 not found"
    }
    assert Task.query.all() == []
```